lib/.tern-port

```diff
@@ -0,0 +1 @@
+42355
```

Comment:
What is this?

Reply:
@nechaido I don't understand this number too )

Reply:
@tshemsedinov this file was committed by mistake, it is no longer there.

Reply:
@nechaido sorry for the noise if it's not relevant anymore, but FWIW I'd suggest adding `.tern-port` to global `.gitignore` if you haven't done so yet.